build.roc

```diff
@@ -0,0 +1,124 @@
+app [main] {
+    cli: platform "TODO use updated basic-cli 0.12.0 (requires changes from github.com/roc-lang/basic-cli/pull/224)",
```

Comment:
Tests are failing because of this btw.
I wanted to remove the dependency on weaver because it would further complicate updating basic-cli on breaking changes.

Reply:
I'll update basic-cli 0.12.0 tomorrow.

Reply:
I wanted to get some other changes in before updating and am blocked on https://github.com/roc-lang/roc/issues/6873, I'm investigating.

Reply:
I posted another reply here before but somehow it's not here :thinking: 
Anyway, I have a probable fix in https://github.com/roc-lang/roc/pull/6881, I just need Ayaz to check it.